.travis/before_script.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 set -ev
 
-mozdownload --type daily --branch mozilla-aurora
+if [[ $LOCALE = 'en-US' ]]
```

Comment:
Actually we wouldn't need this if condition. mozdownload also accepts `--locale en-US`. But anyway I think that this is fine for now. Thanks.